### ISSUE TITLE
-3 spirit for no reason

### ISSUE THREAD

**Issue by kfriend0115** [LightsHope/issues#848]
Northdale, patch 1.12.1

I need help setting my stats back to normal. When I go to my stats page, its showing that my Spirit is being decreased by -3 but I dont have any debuffs on or any equipment lowering my spirit.

character name is Keons

![spiritglitch](https://user-images.githubusercontent.com/42658659/44555378-792fa300-a6fa-11e8-88f0-5670fb9db6e9.png)

**Comment by Godmatik** [comment]
You're probably wearing the off-hand "Tear of Grief" right? From the quest in Darkshore?

It gives +5 int and -3 spirit.

If you don't see those stats, then your cache is out of date. You need to delete your WDB folder with the game closed.

**Comment by kfriend0115** [comment]
I'm not wearing "Tear of Grief", just 2 swords and throwing knives. I've looked through all of my items I have equiped to make sure nothing is taking off spirit but theres nothing. I also deleted my WDB folder with the game closed, opened the game back up and the -3 spirit was still there. :(

**Comment by nathansokoloff** [comment]
I've also noticed this but never bothered to check this repository.

**Comment by kfriend0115** [comment]
Ok after 3 weeks of waiting for responses and sending tickets back and forth with GMs (who were of no help at all), I figured out that the thing causing -3 Spirit was a ring called "Ring of Scorn". Although it was not listed that -3 spirit was one of the effects of wearing this ring, it actually is.... so beware.  I had to take off all of my equipped items to finally figure out that it was the ring causing problems.

Anyways, this must have been changed VERY recently (within the past 24-48 hours) because when I logged back in today, -3 spirit had been added to the list of stats on the ring.... so thank you Lightshope for making me look like an idiot lmfao.

![wowringglitchscreenshot1](https://user-images.githubusercontent.com/42658659/45478324-42511980-b709-11e8-9011-f21de4cfa6ec.PNG)
![wowringglitchscreenshot2](https://user-images.githubusercontent.com/42658659/45478330-45e4a080-b709-11e8-8726-d4188ef29e39.png)
